typos.toml

```diff
@@ -0,0 +1,28 @@
+[files]
+extend-exclude = [
+  "*.pbxproj",    # metadata file
+  "CHANGELOG.md", # To keep consistency between the commit history/PRs.
+]
+
+# Corrections take the form of a key/value pair. The key is the incorrect word
+# and the value is the correct word. If the key and value are the same, the
+# word is treated as always correct. If the value is an empty string, the word
+# is treated as always incorrect.
+
+# Match Whole Word - Case Sensitive
+[default.extend-identifiers]
+ser = "ser" # ron::ser - Serializer
+toi = "toi" # Time of intersention
+Sur = "Sur" # macOS Big Sur - South
+
+# Match Inside a Word - Case Insensitive
+[default.extend-words]
+LOD = "LOD" # Level of detail
+Ot = "Ot"   # Ot - Offset Tangent
```

Comment:
Not sure how often this acronym is used, but it should probably be moved to `extend-identifiers` for the case-sensitivity at least. I imagine "ot" might be a common typo for "it".

Reply:
It is used in the vOt, vOt2 and fLenOt variables in bevy_mikktspace.
I'm going to move them to the extend-ignore-identifiers-re.